### PR TITLE
177 update search bar design

### PR DESCRIPTION
**Pull request by lucasliebe**
This Pull Request will change the visuals of the search page to a more cleaner look. A Search can now be started by pressing enter and cleared using the cross, that is only visible when text is entered.
image

### CODE REVIEW

**Review by kpostnov — CHANGES_REQUESTED** [review]
There is a couple of issues I have with that PR so far:

- The page looks very empty now, both on desktop and mobile. Someone using the website for the first time might not know what the site is for and what to do. Maybe a short introductory text could be useful.
- I believe there is a nicer "x" that isn't just the actual character and might look more appealing. The "x" could use a white background as well for better readability when writing long queries.
- I feel like there should be another way than reloading the page to clear the input.

Bootstrap should have a predefined design for input forms with icons. Something like this: https://stackoverflow.com/questions/45696685/search-input-with-an-icon-bootstrap

File: app/assets/stylesheets/search.scss [hunk 1]
```diff
@@ -34,3 +34,24 @@
     }
   }
 }
+
+.cancel-input{
```
Comment (treyfel) [inline app/assets/stylesheets/search.scss:38]: maybe add a border radius here

**Comment by FranzSw** [comment]
I agree with Kirill.

> The page looks very empty now, both on desktop and mobile. Someone using the website for the first time might not know what the site is for and what to do. Maybe a short introductory text could be useful.

I would keep the search bar at top, maybe short intro text below?
Also a placeholder in the field might already be enough ("Suche nach Räumen, Personen, ...")

> I feel like there should be another way than reloading the page to clear the input.

Imho, the results dont have to be cleared when clicking the X, that might make it a bit easier?

**Comment by kpostnov** [comment]
> I would keep the search bar at top, maybe the short intro text below? Also a placeholder in the field might already be enough ("Suche nach Räumen, Personen, ...")

Apparently there is another issue for that (#213). But the placeholder sounds good.

> Imho, the results dont have to be cleared when clicking the X, that might make it a bit easier?

I agree. Usually it's done this way.

**Comment by lucasliebe** [comment]
I now changed some of your recommendations the rest was cleared in discussion with the PO

**Review by kpostnov — DISMISSED** [review]
# Looks not fine to me now

**Review by kpostnov — APPROVED** [review]

**Pull request merged**